testdata/expected/src/github.com/sgtest/css-sample-0/css-sample-0/Dir.graph.json

```diff
@@ -0,0 +1,528 @@
+{
+  "Defs": [
+    {
+      "UnitType": "Dir",
+      "Unit": "css-sample-0",
+      "Path": "#container",
```

Comment:
It might clash, if so, we should be using, for example `encodeURIComponent` on the browser.

We do so in sourcegraph when creating links to definitions, for example:
```
DefBackend.fetch(`/.api/repos/${action.repo}${action.rev ? `@${action.rev}` : ""}/-/def/${encodeURIComponent(action.def)}`)
							.then(updateRepoCloning(action.repo))
							.then(checkStatus)
							.then((resp) => resp.json())
							.catch((err) => ({Error: err}))
							.then((data) => Dispatcher.Stores.dispatch(new DefActions.DefFetched(action.repo, action.rev, action.def, data)))
```

Reference: https://sourcegraph.com/sourcegraph/sourcegraph@html-to-css-jump-to-def/-/blob/app/web_modules/sourcegraph/def/DefBackend.js#L21